locale/localeconv.c

```diff
@@ -25,42 +25,45 @@ __localeconv (void)
 {
   static struct lconv result;
 
-  result.decimal_point = (char *) _NL_CURRENT (LC_NUMERIC, DECIMAL_POINT);
-  result.thousands_sep = (char *) _NL_CURRENT (LC_NUMERIC, THOUSANDS_SEP);
-  result.grouping = (char *) _NL_CURRENT (LC_NUMERIC, GROUPING);
-  if (*result.grouping == '\177' || *result.grouping == '\377')
-    result.grouping = (char *) "";
+  // Qianxi Edit: locale related stuff is not working currently
+  // this feature is not important so we might look into this later
+  // if we want to support it in the future
+//   result.decimal_point = (char *) _NL_CURRENT (LC_NUMERIC, DECIMAL_POINT);
+//   result.thousands_sep = (char *) _NL_CURRENT (LC_NUMERIC, THOUSANDS_SEP);
+//   result.grouping = (char *) _NL_CURRENT (LC_NUMERIC, GROUPING);
+//   if (*result.grouping == '\177' || *result.grouping == '\377')
+//     result.grouping = (char *) "";
 
-  result.int_curr_symbol = (char *) _NL_CURRENT (LC_MONETARY, INT_CURR_SYMBOL);
-  result.currency_symbol = (char *) _NL_CURRENT (LC_MONETARY, CURRENCY_SYMBOL);
-  result.mon_decimal_point = (char *) _NL_CURRENT (LC_MONETARY,
-						   MON_DECIMAL_POINT);
-  result.mon_thousands_sep = (char *) _NL_CURRENT (LC_MONETARY,
-						   MON_THOUSANDS_SEP);
-  result.mon_grouping = (char *) _NL_CURRENT (LC_MONETARY, MON_GROUPING);
-  if (*result.mon_grouping == '\177' || *result.mon_grouping == '\377')
-    result.mon_grouping = (char *) "";
-  result.positive_sign = (char *) _NL_CURRENT (LC_MONETARY, POSITIVE_SIGN);
-  result.negative_sign = (char *) _NL_CURRENT (LC_MONETARY, NEGATIVE_SIGN);
+//   result.int_curr_symbol = (char *) _NL_CURRENT (LC_MONETARY, INT_CURR_SYMBOL);
+//   result.currency_symbol = (char *) _NL_CURRENT (LC_MONETARY, CURRENCY_SYMBOL);
+//   result.mon_decimal_point = (char *) _NL_CURRENT (LC_MONETARY,
+// 						   MON_DECIMAL_POINT);
+//   result.mon_thousands_sep = (char *) _NL_CURRENT (LC_MONETARY,
+// 						   MON_THOUSANDS_SEP);
+//   result.mon_grouping = (char *) _NL_CURRENT (LC_MONETARY, MON_GROUPING);
+//   if (*result.mon_grouping == '\177' || *result.mon_grouping == '\377')
```

Comment:
similar here.  Why comment this out?

Reply:
Also added as comment: locale related stuff is not working currently since a lot of locale symbol is not found by wasm-ld. I currently havn't looked into the root reason for this, but this feature does not seem to be important right now so we might look into this later if we want to support it in the future